### PR TITLE
Add HTTP-API support

### PR DESCRIPTION
**Pull request by reycn**
**Changes:**
[x] Add HTTP-API support, with a simple format notice;
[x] Updated `requirements.txt`: `fastapi` added.
[x] Updated `READ.ME`: HTTP-API tutorial, including how to install `uvicron`.
[x] Local & remote nginx reverse-proxy test passed.

**e.g.**
```
IN: 
http(s)://host:port/how%20are%20you

OUT:
"I'm a machine learning model trained by OpenAI, so I don't have feelings or emotions like a human does. I'm designed to provide answers to questions to the best of my ability based on the information I've been trained on. Is there something specific you would like to know?”
```

**On login&captcha**: 
Directly use the session token stored by cli and currently no expiration test implemented. This is because, if the token was expired, run the main cli would be enough.

### CODE REVIEW

File: webapi.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                                      previous_convo_id=AI_PREV_CONVO)
+    if answer == "400" or answer == "401":
+        result = ">> Token invalid, please contact the owner to refresh."
+        print(f"{Fore.RED}>> Your token is invalid. Attempting to refresh..")
+        pass
+    else:
+        result = answer
+        if previous_convo is not None:
+            AI_PREV_CONVO = previous_convo
+    print(result)
```
Comment (rawandahmad698) [inline webapi.py:47]: And remove this

**Review by rawandahmad698 — CHANGES_REQUESTED** [review]

**Comment by reycn** [comment]
Sure, working on it.

**Comment by rawandahmad698** [comment]
Thank you.

**Comment by reycn** [comment]
- Functions, variables renamed;
- Exceptions raised as ValueError;
- Error msgs changed as instructed.

**Comment by rawandahmad698** [comment]
Looking good, will merge after tests

**Comment by daymanvt** [comment]
Appreciate you guys working on this.  Maybe in a few hours of using chatgpt, I can even start helping :)

**Comment by reycn** [comment]
A kindly notice: some special characters might hamper the simple solution, use post instead to solve the problem.

I will add this to a new version in few days. Currently busy for some personal issues. :0

**Comment by rawandahmad698** [comment]
Sorry for the delay, Since I released v1.0.2, this code would be conflicting, could you integrate it with the new library?

**Comment by reycn** [comment]
> Sorry for the delay, Since I released v1.0.2, this code would be conflicting, could you integrate it with the new library?

OK but I need one or two days since busy for some important issues.

**Pull request closed without merging**